src/config.c

```diff
@@ -1101,7 +1102,7 @@ struct rewriteConfigState *rewriteConfigReadOldFile(char *path) {
         /* If the config file contains a configuration that is no longer present
          * in the configs array: remove it from the .conf file. Use case right now
          * is module with configurations unloaded after a rewrite with the module configs. */
-        if (!lookupConfig(argv[0])) {
+        if (strcasecmp(argv[0], "sentinel") && !lookupConfig(argv[0])) {
```

Comment:
maybe we can check `server.sentinel_mode` instead of that string compare?
i.e. disable that feature in sentinel mode completely?

Reply:
I'm not altogether too familiar with sentinel mode. It wouldn't be the most difficult thing in the world to change this logic to check argv[1] instead if sentinel mode is enabled. Will leave it to your discretion here.